.eslintrc.js

```diff
@@ -7,4 +7,10 @@ module.exports = {
       },
     },
   },
+  rules: {
+    '@typescript-eslint/camelcase': 'off',
```

Comment:
Is that needed for the `Koa` type?

Reply:
Nope, for third party API response like `access_token`